v4-encryption/src/main/java/com/adobe/platform/core/identity/services/cosmosdb/client/V4EncryptionCosmosDbClient.java

```diff
@@ -348,7 +348,7 @@ private static IndexingPolicy getDefaultIndexingPolicy() {
 		ClientEncryptionIncludedPath includedPath = new ClientEncryptionIncludedPath();
 		includedPath.setPath("/" + ENCRYPTED_FIELD);
 		includedPath.setEncryptionType(CosmosEncryptionType.DETERMINISTIC);
-		includedPath.setEncryptionAlgorithm(CosmosEncryptionAlgorithm.AEAES_256_CBC_HMAC_SHA_256);
```

Comment:
why change the algorithm?

Reply:
In the latest encryption project, encryption team has changed it, so updating it here :)